Enyim.Caching/Memcached/MemcachedNode.cs

```diff
@@ -387,6 +394,95 @@ internal async Task InitPoolAsync()
                 }
             }
 
+            private void StartReconciliationTask()
+            {
+                if (_connectionIdleTimeout == TimeSpan.Zero)
+                    return;
+
+                var reconcileTimer = new PeriodicTimer(_connectionIdleTimeout);
+                _ = RunTimer();
+
+                async Task RunTimer()
+                {
+                    while (await reconcileTimer.WaitForNextTickAsync().ConfigureAwait(false))
+                    {
+                        try
+                        {
+                            using var source = new CancellationTokenSource(_connectionIdleTimeout);
+                            await reconcileAsync(source.Token).ConfigureAwait(false);
+                        }
+                        catch(Exception e)
+                        {
+                            _logger.LogWarning("ReconciliationTaskFailed", new EventId(0), e);

```

Comment:
Should this be an Error?

Reply:
I dont think so, The StartReconciliationTask is a background task, and its failure will not impact the current flow of the system